### ISSUE TITLE
Template has ORIGIN_HOSTNAME check that breaks wrangler

### ISSUE THREAD

**Issue by martonfekete** [adobe/helix-cloudflare-prod-worker-template#6]
### Expected Behaviour
Template works without issues when all environment variables are set correctly.

### Actual Behaviour
Default `index.mjs` from template breaks with the following errors:

```
[wrangler:err] TypeError: url.match is not a function
    at Object.handleRequest [as fetch] (file:///src/index.mjs:40:12)
    at fetchDispatcher (file:///.wrangler/tmp/bundle-rIuIXO/middleware-loader.entry.ts:56:17)
```

#### Steps to Reproduce
1. Create repository from template
2. Update `wrangler.toml` with the correct `ORIGIN_HOSTNAME `
3. Run the default script with `wrangler dev`.

**Comment by martonfekete** [comment]
I also created a PR to fix the issue
https://github.com/adobe/helix-cloudflare-prod-worker-template/pull/7

**Comment by stefan-guggisberg** [comment]
resolved via #7
